GeoFire.xcodeproj/project.pbxproj

```diff
@@ -22,19 +22,19 @@
 /* End PBXAggregateTarget section */
 
 /* Begin PBXBuildFile section */
+		38FC6CB01CFBE57C002AB722 /* GeoFireTest.m in Sources */ = {isa = PBXBuildFile; fileRef = FB8C8845196D55630061AC0F /* GeoFireTest.m */; };
```

Comment:
yeah sorry, I don't use pods much noticed it on the weekned, havent had a chance to fix.